README.md

```diff
@@ -1,5 +1,7 @@
 # Charmed Ory Hydra
 
+[![CharmHub Badge](https://charmhub.io/hydra/badge.svg)](https://charmhub.io/hydra)
+
```

Comment:
we could add the github bagde as well
```suggestion
[![CI](https://github.com/canonical/hydra-operator/actions/workflows/on_push.yaml/badge.svg)](https://github.com/canonical/hydra-operator/actions/workflows/on_push.yaml)

```

Reply:
i agree with  @natalian98 but we can probably move on and get that done in a separate batch of PRs